From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 6 Mar 2019 18:26:41 +0100
Subject: magnifier: Fix color argument

Clutter.TextureNode takes a Clutter.Color, not a Cogl.Color.

https://gitlab.gnome.org/GNOME/gnome-shell/issues/1020

Origin: upstream, commit:de0fd86602d3f7f6c001f7ee05e3cfc364a35aa
Applied-Upstream: 3.32.1
---
 js/ui/magnifier.js | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/js/ui/magnifier.js b/js/ui/magnifier.js
index 753151a..546ab1d 100644
--- a/js/ui/magnifier.js
+++ b/js/ui/magnifier.js
@@ -1,6 +1,6 @@
 // -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
 
-const { Atspi, Clutter, Cogl, GDesktopEnums,
+const { Atspi, Clutter, GDesktopEnums,
         Gio, GLib, GObject, Meta, Shell, St } = imports.gi;
 const Mainloop = imports.mainloop;
 const Signals = imports.signals;
@@ -65,9 +65,7 @@ var MouseSpriteContent = GObject.registerClass({
         if (!this._texture)
             return;
 
-        let color = new Cogl.Color();
-        color.init_from_4ub(0, 0, 0, 0);
-
+        let color = new Clutter.Color();
         let textureNode = new Clutter.TextureNode(this._texture,
                                                   color,
                                                   Clutter.ScalingFilter.NEAREST,
